package.json

```diff
@@ -87,7 +90,7 @@
     "test:rule-help-version": "mocha test/test-rule-help-version.js",
     "test:node": "mocha test/node/*.js",
     "version": "echo \"use 'npm run release' to bump axe-core version\" && exit 1",
-    "release": "standard-version -a",
+    "release": "git fetch origin --tags --force && standard-version -a",
```

Comment:
Why `--force`?

Reply:
https://stackoverflow.com/a/55268620/2124254

Reply:
For now anyone whose done the release script will have incorrect tags, so best we solve that problem too